src/main/java/org/craftercms/studio/impl/v1/util/scripts/StudioScriptingClassLoader.java

```diff
@@ -23,7 +23,7 @@ public Class<?> loadClass(String name) throws ClassNotFoundException {
         Class<?> toRet = null;
         try {
             toRet = super.loadClass(name);
-        } catch (ClassNotFoundException exc) {
+        } catch (ClassNotFoundException e) {
 
```

Comment:
Add a debug log statement instead of silently swallowing the exception?

Reply:
Class deleted.